builds/android/app/src/main/java/org/easyrpg/player/player/EasyRpgPlayerActivity.java

```diff
@@ -354,11 +354,7 @@ public Game getProjectPath() {
      * @return Full path to the RTP
      */
     public String getRtpPath() {
-        // TODO : Return RTP
-        return "";
-        //String str = SettingsManager.getEasyRPGFolder() + "/rtp";
-        // Log.v("SDL", "getRtpPath " + str);
-        //return str;
+        return SettingsManager.getRtpFolder().toString();
```

Comment:
will this return a valid SAF Uri? (content:// and stuff)

Reply:
Yep :)

Reply:
I just tested it.
You must use ``return SettingsManager.getRtpFolder().getUri().toString();``, then it will work.Yours gives back the class name + pointer (Standard toString impl)